realm-transformer/src/test/groovy/io/realm/transformer/BytecodeModifierTest.groovy

```diff
@@ -98,18 +98,22 @@ class BytecodeModifierTest extends Specification {
         then: 'the field is not used in the method anymore'
         def methodInfo = ctMethod.getMethodInfo()
         def codeAttribute = methodInfo.getCodeAttribute()
+        def methodCalled = false
         def fieldIsUsed = false
         for (CodeIterator ci = codeAttribute.iterator(); ci.hasNext();) {
             int index = ci.next();
             int op = ci.byteAt(index);
             if (op == Opcode.GETFIELD) {
                 fieldIsUsed = true
+            } else if (op == Opcode.INVOKEVIRTUAL) {
+                // access to the getter
+                methodCalled = true
```

Comment:
We replace a field access with method call.
I think it's better to directly check whether there is a method call (`INVOKEVIRTUAL`) or not.

Another reason is to align all tests below those all have a check for method call.